<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QLabel
{
	qproperty-alignment:'AlignVCenter|AlignRight';
	border:1px solid gray;
}
background-color:white;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>241</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	qproperty-alignment:'AlignVCenter|AlignRight';
	border:1px solid gray;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Plus_Divide_Sign">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Percent_Sign">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Add_Sign">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton
{
rgb(255, 170, 0)
qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 255), stop:0.19397 rgba(0, 0, 0, 255), stop:0.202312 rgba(122, 97, 0, 255), stop:0.495514 rgba(76, 58, 0, 255), stop:0.504819 rgba(255, 255, 255, 255), stop:0.79 rgba(255, 255, 255, 255), stop:1 rgba(255, 158, 158, 255))
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Mutiply">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Minus_Sign">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Divide">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Key_Point">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Q PushButton
{
border:1px solid gray;
}
QPushButton:pressed
{
background-color:qlineargradient(x1:0;y1:0,x2:0;y2:1;stop:0#dadbde,stop:1#f6f7fa);
}


</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Equal_Sign">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
